admin-dev/themes/new-theme/scss/components/_form.scss

```diff
@@ -74,4 +74,9 @@
       outline: none;
     }
   }
+
+  //@todo no idea if adding new class is a good idea, I needed class for form value, add same padding as label
```

Comment:
ping @NeOMakinG for advice

Reply:
It's fine, prefer adding a new class for custom styles not linked to Bootstrap or the UIKit